interfaces/cython/cantera/transport.pyx

```diff
@@ -256,6 +260,79 @@ cdef class Transport(_SolutionBase):
         def __get__(self):
             return get_transport_1d(self, tran_getMobilities)
 
+    def viscosity_polynomial(self, i):
+        """Get the polynomial fit to the logarithm of temperature for 
+        the viscosity of species i."""
+        cdef np.ndarray[np.double_t, ndim=1] data = np.empty(4 if self.transport.CKMode() else 5)
+        tran_getViscosityPolynomials(self.transport, i, &data[0])
+        return data
+
+    def thermal_conductivity_polynomial(self, i):
+        """Get the polynomial fit to the logarithm of temperature for 
+        the thermal conductivity of species i."""
+        cdef np.ndarray[np.double_t, ndim=1] data = np.empty(4 if self.transport.CKMode() else 5)
+        tran_getConductivityPolynomials(self.transport, i, &data[0])
+        return data
+
+    def binary_diff_coeffs_polynomial(self, i, j):
+        """Get the polynomial fit to the logarithm of temperature for 
+        the binary diffusion coefficient of species i and j."""
+        cdef np.ndarray[np.double_t, ndim=1] data = np.empty(4 if self.transport.CKMode() else 5)
+        tran_getBinDiffusivityPolynomials(self.transport, i, j, &data[0])
+        return data
+
+    def collision_integral_polynomials(self, i, j):
+        """Get the polynomial fit to the logarithm of temperature for 
+        the collision integral of species i and j."""
+        cdef np.ndarray[np.double_t, ndim=1] adata = np.empty(7 if self.transport.CKMode() else 9)
+        cdef np.ndarray[np.double_t, ndim=1] bdata = np.empty(7 if self.transport.CKMode() else 9)
+        cdef np.ndarray[np.double_t, ndim=1] cdata = np.empty(7 if self.transport.CKMode() else 9)
+        tran_getCollisionIntegralPolynomials(self.transport, i, j, &adata[0], &bdata[0], &cdata[0])
+        return adata, bdata, cdata
+
+    def modify_viscosity_polynomial(self, i, values):
```

Comment:
For getters/setters, I would recommend the more commonly used prefixes `get_` and `set_`. More often than not, Cantera omits prefixes only when accessing `properties` (which don't use function arguments).

Reply:
Done